## 37% of end-end tests have passed
```
Total executed tests: 49
Failed tests: 30
Expected to fail tests: 1
Passed tests: 18
Not executed tests: 2
```
```
IDs of not executed tests: 727229,727230 
```
```
IDs of failed tests: 702845,702840,702894,703086,702855,702947,702958,702775,702786,702731,702844,704613,702841,702839,702733,702809,702838,704615,702948,702859,703629,703133,702957,702843,702850,703391,702732,703503,703495,703194 
```
```
IDs of expected to fail tests: 727232 
```
<h3>Not executed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/remove-non-operable-accounts-in-joining-a-community&PR_ID=20636&APK_NAME=20636.apk&TR_CASE_IDS=727229,727230">Rerun not executed tests</a></li></details><h3>Failed tests (30)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/remove-non-operable-accounts-in-joining-a-community&PR_ID=20636&APK_NAME=20636.apk&TR_CASE_IDS=702845,702840,702894,703086,702855,702947,702958,702775,702786,702731,702844,704613,702841,702839,702733,702809,702838,704615,702948,702859,703629,703133,702957,702843,702850,703391,702732,703503,703495,703194">Rerun failed tests</a></li><br/><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601948">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601947">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` </div><div>Device 2: Wait for element `Button` for max 120s and click when it is available </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9560522aea847c38b16e8177cfbcf1e?auth=bd97118010c953a2f18ac01cae11ccf1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51a019eb16104443b4dcce98a60e6b42?auth=3c7673787bc62b28806242a7bf442f40">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601937">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601926">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` </div><div>Device 2: Wait for element `Button` for max 120s and click when it is available </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a002d1564784be28136be381645e519?auth=54532f7dd10d3ead9f36fbb55189857b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/42e0614930c14ccdbb6c0424bf76b4a9?auth=a590fc5105bf00c5fe0a1e039e471f5b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601929">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601946">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601959">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601941">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:378: in test_1_1_chat_edit_message
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#365">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601916">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:198: in test_1_1_chat_pin_messages
    chat.pinned_messages_count.wait_for_element_text(text="2",
../views/base_element.py:190: in wait_for_element_text
    self.driver.fail(message if message else "`%s` is not equal to expected `%s` in %s sec" % (
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Pinned messages count is not 2 as expected!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#217">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61#175">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601918">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:470: in test_1_1_chat_text_message_delete_push_disappear
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:175: in _element_if_visible
    return element if element.is_displayed() == visibility else False
 'str' object has no attribute 'is_displayed'
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#370">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601955">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:405: in test_1_1_chat_send_image_save_and_share
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:175: in _element_if_visible
    return element if element.is_displayed() == visibility else False
 'str' object has no attribute 'is_displayed'
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#368">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601950">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `show-profiles` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:240: in test_restore_multiaccount_with_waku_backup_remove_switch
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 zQ3...dWXh5 was not restored as a contact from waku backup!
E    zQ3...Vacac was not restored as a contact from waku backup!
E    admin_open was not restored from waku-backup!!
E    member_open was not restored from waku-backup!!
E    admin_closed was not restored from waku-backup!!
E    member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c583a85252549cca92ad7080522cbc5?auth=127242e7bf6f7310b3faee7f442c2ca8#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601958">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `communities-home-discover-card` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:261: in test_community_discovery
    self.home.community_card_item.wait_for_elements(seconds=120)
../views/base_element.py:138: in wait_for_elements
    raise TimeoutException(
 Device 1:  BaseElement by accessibility id:`community-card-item` is not found on the screen after wait_for_elements
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c583a85252549cca92ad7080522cbc5?auth=127242e7bf6f7310b3faee7f442c2ca8#378">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601933">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601944">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601949">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601936">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601934">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601932">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601931">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` </div><div>Device 2: Wait for element `Button` for max 120s and click when it is available </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdcab2e6165e4584b2e1cf903e5b27be?auth=5ead7b56e787f47cc13011853380b485">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5737ed020f514cc491ae012ab3abd357?auth=5d4294fb1c7ddaa52925422dc093999b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601961">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601942">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601935">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601951">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:347: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601917">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
critical/chats/test_group_chat.py:354: in test_group_chat_pin_messages
    [self.homes[i].navigate_back_to_home_view() for i in range(3)]
critical/chats/test_group_chat.py:354: in <listcomp>
    [self.homes[i].navigate_back_to_home_view() for i in range(3)]
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9bfcc152e884baf91dfe18eccf0bd44?auth=9e781b89d0efc119c89727e5e94120f0#359">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c25c8432f6e4448a44a53455c29c240?auth=cbe638e10c40acdc6f5eba5d0bb8b4e9#232">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9679935e30744f3da0901f72a18bf892?auth=9b5fabc95d5ae64e4aed4b14fb2c5fb1#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601956">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div>Device 1: Could not reach home view by pressing system back button </div><div>Device 2: Could not reach home view by pressing system back button </div></blockquote></p>

```
critical/chats/test_group_chat.py:439: in test_group_chat_mute_chat
    [self.homes[i].navigate_back_to_home_view() for i in range(3)]
critical/chats/test_group_chat.py:439: in <listcomp>
    [self.homes[i].navigate_back_to_home_view() for i in range(3)]
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9bfcc152e884baf91dfe18eccf0bd44?auth=9e781b89d0efc119c89727e5e94120f0#368">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c25c8432f6e4448a44a53455c29c240?auth=cbe638e10c40acdc6f5eba5d0bb8b4e9#241">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9679935e30744f3da0901f72a18bf892?auth=9b5fabc95d5ae64e4aed4b14fb2c5fb1#231">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601939">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `close-activity-center` </div><div>Device 2: Tap on found: Button </div></blockquote></p>

```
activity_center/test_activity_center.py:95: in test_activity_center_contact_request_decline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Pending contact request is not shown on unread notification element on Activity center!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7a5b9087ba0a4f278c886d3028d37d45?auth=f3f7ef253006a692a1b1afb557c1c413#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0561fba9caf044e5920221eda96b061d?auth=d43ec0dc0fa6c7ef45b2b6820e0932fd#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601945">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` </div><div>Device 2: Wait for element `Button` for max 120s and click when it is available </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:274: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b6cb850c18494a6993345c0733d708a5?auth=dcf6be98e2b6feb7b128926f0e778540">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a9de8912266840f4b3b5dff1ae8037e1?auth=ed758f0cf17b3278645205f360f2aa47">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601921">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `BrowserTab` by `accessibility id`: `browser-stack-tab` </div><div>Device 1: Tap on found: BrowserTab </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:115: in test_links_deep_links
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'token-gated community e2e' was not requested to join by the deep link status.app://c/G0IAAGS9TbI9SrSPbDPplfBUeBBOIWhFRhGIKVlxntAg5CSvsJw4jurb636UKo7Lm2gTUS_M#zQ3shm7cS2MX6K1iinjNhiw88g1WH6fbE8rvnMHBq4RhznkWc
E    Community 'open community' was not requested to join by the deep link status.app://c/G0UAAMTyNsn2QZDEG0EXftOl8pOEfwEBOOSA_YTfIk85xmADDgINGmxpUHAXzK36bN0fK42Xf4YD2yjPk1z2pbFwFw==#zQ3shZU9PNP6QHwEmqoh5mQqjCMDsfZYnpY4BEC8hom7KoNEz
E    Community 'closed community' was not requested to join by the deep link status.app://c/G00AAGS9TbI9mSR-ZNmFrhRjNuEeXAAbcAIUaLLJyjMOG3ACJQ12oIHD78QhzO9s_T5bUeU7rnATWJg3mGgTUemrAg==#zQ3shTK1zXmLq2yZWMij65j1LNtmvuG6x71Nmg2fwtKs32wHj
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc45b40bfdeb4aa9a05341cbc977995d?auth=bcce334e035ec21cbfc9408d24ee62ab#214">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601960">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for chat: 'general' </div><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'token-gated community e2e' was not requested to join by the url https://status.app/c/G0IAAGS9TbI9SrSPbDPplfBUeBBOIWhFRhGIKVlxntAg5CSvsJw4jurb636UKo7Lm2gTUS_M#zQ3shm7cS2MX6K1iinjNhiw88g1WH6fbE8rvnMHBq4RhznkWc
E    Community 'open community' was not requested to join by the url https://status.app/c/G0UAAMTyNsn2QZDEG0EXftOl8pOEfwEBOOSA_YTfIk85xmADDgINGmxpUHAXzK36bN0fK42Xf4YD2yjPk1z2pbFwFw==#zQ3shZU9PNP6QHwEmqoh5mQqjCMDsfZYnpY4BEC8hom7KoNEz
E    Community 'closed community' was not requested to join by the url https://status.app/c/G00AAGS9TbI9mSR-ZNmFrhRjNuEeXAAbcAIUaLLJyjMOG3ACJQ12oIHD78QhzO9s_T5bUeU7rnATWJg3mGgTUemrAg==#zQ3shTK1zXmLq2yZWMij65j1LNtmvuG6x71Nmg2fwtKs32wHj
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc45b40bfdeb4aa9a05341cbc977995d?auth=bcce334e035ec21cbfc9408d24ee62ab#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601965">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EditBox` by `accessibility id`: `add-address-to-watch` </div><div>Device 1: Type `0x8d2413447ff297d30bdc475f6d5cb00254685aae` to `EditBox` </div></blockquote></p>

```
critical/test_wallet.py:237: in test_wallet_add_remove_watch_only_account
    self.wallet_view.add_watch_only_account(address=address_to_watch, account_name=new_account_name)
../views/wallet_view.py:150: in add_watch_only_account
    self.account_has_activity_label.wait_for_visibility_of_element()
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: Text by accessibility id:`account-has-activity` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cb26cdec4b7048bfb0a01da13634067e?auth=2aeb4870231dbee7d5cef6a2abe81ba2#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (18)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601964">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cb26cdec4b7048bfb0a01da13634067e?auth=2aeb4870231dbee7d5cef6a2abe81ba2#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601957">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3549dd1b5a9a40f08b080e356ace0fdf?auth=43383fe8aa4c16afc736d74ae428aee5#114">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f3e9cd89798a448d825ed33dd73e4b29?auth=517e41436c9fb25e0ae17f7bb595f622#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601924">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3549dd1b5a9a40f08b080e356ace0fdf?auth=43383fe8aa4c16afc736d74ae428aee5#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f3e9cd89798a448d825ed33dd73e4b29?auth=517e41436c9fb25e0ae17f7bb595f622#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601925">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3549dd1b5a9a40f08b080e356ace0fdf?auth=43383fe8aa4c16afc736d74ae428aee5#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f3e9cd89798a448d825ed33dd73e4b29?auth=517e41436c9fb25e0ae17f7bb595f622#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601915">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601920">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#258">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61#199">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601930">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#316">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61#249">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601923">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88c61b08f657475e969aae7cfcf50b6c?auth=6e879dfa04f424b004a6d5bb682beb14#151">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19eebe7217d4404dac3f3b662219043d?auth=28c62b63db229c32432973dbf3e99a61#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601919">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c583a85252549cca92ad7080522cbc5?auth=127242e7bf6f7310b3faee7f442c2ca8#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601938">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c583a85252549cca92ad7080522cbc5?auth=127242e7bf6f7310b3faee7f442c2ca8#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601943">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c583a85252549cca92ad7080522cbc5?auth=127242e7bf6f7310b3faee7f442c2ca8#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601954">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c583a85252549cca92ad7080522cbc5?auth=127242e7bf6f7310b3faee7f442c2ca8#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601952">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9bfcc152e884baf91dfe18eccf0bd44?auth=9e781b89d0efc119c89727e5e94120f0#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c25c8432f6e4448a44a53455c29c240?auth=cbe638e10c40acdc6f5eba5d0bb8b4e9#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9679935e30744f3da0901f72a18bf892?auth=9b5fabc95d5ae64e4aed4b14fb2c5fb1#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601927">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9bfcc152e884baf91dfe18eccf0bd44?auth=9e781b89d0efc119c89727e5e94120f0#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c25c8432f6e4448a44a53455c29c240?auth=cbe638e10c40acdc6f5eba5d0bb8b4e9#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9679935e30744f3da0901f72a18bf892?auth=9b5fabc95d5ae64e4aed4b14fb2c5fb1#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601928">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9bfcc152e884baf91dfe18eccf0bd44?auth=9e781b89d0efc119c89727e5e94120f0#322">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c25c8432f6e4448a44a53455c29c240?auth=cbe638e10c40acdc6f5eba5d0bb8b4e9#205">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9679935e30744f3da0901f72a18bf892?auth=9b5fabc95d5ae64e4aed4b14fb2c5fb1#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601953">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9bfcc152e884baf91dfe18eccf0bd44?auth=9e781b89d0efc119c89727e5e94120f0#282">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c25c8432f6e4448a44a53455c29c240?auth=cbe638e10c40acdc6f5eba5d0bb8b4e9#170">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9679935e30744f3da0901f72a18bf892?auth=9b5fabc95d5ae64e4aed4b14fb2c5fb1#141">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601940">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7a5b9087ba0a4f278c886d3028d37d45?auth=f3f7ef253006a692a1b1afb557c1c413#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0561fba9caf044e5920221eda96b061d?auth=d43ec0dc0fa6c7ef45b2b6820e0932fd#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3601922">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7a5b9087ba0a4f278c886d3028d37d45?auth=f3f7ef253006a692a1b1afb557c1c413#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0561fba9caf044e5920221eda96b061d?auth=d43ec0dc0fa6c7ef45b2b6820e0932fd#208">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/communities/actions/accounts_selection/events.cljs
+++ b/src/status_im/contexts/communities/actions/accounts_selection/events.cljs
@@ -18,7 +18,7 @@
 
 (defn do-init-permission-addresses
   [{:keys [db]} [community-id revealed-accounts]]
-  (let [wallet-accounts     (utils/sorted-non-watch-only-accounts db)
+  (let [wallet-accounts     (utils/sorted-operable-non-watch-only-accounts db)
         addresses-to-reveal (if (seq revealed-accounts)
                               (set (keys revealed-accounts))
                               ;; Reveal all addresses as fallback.
@@ -62,7 +62,7 @@
   status-go will default to all available."
   [{:keys [db]} [{:keys [community-id password on-success addresses airdrop-address]}]]
   (let [pub-key             (get-in db [:profile/profile :public-key])
-        wallet-accounts     (utils/sorted-non-watch-only-accounts db)
+        wallet-accounts     (utils/sorted-operable-non-watch-only-accounts db)
         addresses-to-reveal (if (seq addresses)
                               (set addresses)
                               (get-in db [:communities/all-addresses-to-reveal community-id]))

--- a/src/status_im/contexts/communities/actions/accounts_selection/events_test.cljs
+++ b/src/status_im/contexts/communities/actions/accounts_selection/events_test.cljs
@@ -9,17 +9,25 @@
 (def wallet-accounts
   {"0xA" {:address     "0xA"
           :watch-only? true
+          :operable?   true
           :position    2
           :color       :red
           :emoji       "🦇"}
-   "0xB" {:address  "0xB"
-          :position 0
-          :color    :blue
-          :emoji    "🐈"}
-   "0xC" {:address  "0xC"
-          :position 1
-          :color    :orange
-          :emoji    "🛏️"}})
+   "0xB" {:address   "0xB"
+          :operable? true
+          :position  0
+          :color     :blue
+          :emoji     "🐈"}
+   "0xC" {:address   "0xC"
+          :operable? true
+          :position  1
+          :color     :orange
+          :emoji     "🛏️"}
+   "0xD" {:address   "0xD"
+          :operable? false
+          :position  3
+          :color     :flamingo
+          :emoji     "🦩"}})
 
 (def permissioned-accounts
   [{:address          "0xB"

--- a/src/status_im/contexts/communities/actions/accounts_selection/view.cljs
+++ b/src/status_im/contexts/communities/actions/accounts_selection/view.cljs
@@ -22,7 +22,7 @@
         airdrop-account (rf/sub [:communities/airdrop-account id])
         revealed-accounts (rf/sub [:communities/accounts-to-reveal id])
         revealed-accounts-count (count revealed-accounts)
-        wallet-accounts-count (count (rf/sub [:wallet/accounts-without-watched-accounts]))
+        wallet-accounts-count (count (rf/sub [:wallet/operable-accounts-without-watched-accounts]))
         addresses-shared-text (if (= revealed-accounts-count wallet-accounts-count)
                                 (i18n/label :t/all-addresses)
                                 (i18n/label-pluralize

--- a/src/status_im/contexts/communities/actions/addresses_for_permissions/events.cljs
+++ b/src/status_im/contexts/communities/actions/addresses_for_permissions/events.cljs
@@ -120,7 +120,7 @@
 (defn set-permissioned-accounts
   [{:keys [db]} [community-id addresses-to-reveal]]
   (let [addresses-to-reveal     (set addresses-to-reveal)
-        wallet-accounts         (utils/sorted-non-watch-only-accounts db)
+        wallet-accounts         (utils/sorted-operable-non-watch-only-accounts db)
         current-airdrop-address (get-in db [:communities/all-airdrop-addresses community-id])
         new-airdrop-address     (if (contains? addresses-to-reveal current-airdrop-address)
                                   current-airdrop-address
@@ -142,7 +142,7 @@
   [{:keys [db]} [community-id new-value]]
   (let [current-addresses   (get-in db [:communities/all-addresses-to-reveal community-id])
         addresses-to-reveal (if new-value
-                              (->> (utils/sorted-non-watch-only-accounts db)
+                              (->> (utils/sorted-operable-non-watch-only-accounts db)
                                    (map :address)
                                    set)
                               current-addresses)]

--- a/src/status_im/contexts/communities/actions/addresses_for_permissions/events_test.cljs
+++ b/src/status_im/contexts/communities/actions/addresses_for_permissions/events_test.cljs
@@ -33,9 +33,15 @@
     (let [cofx
           {:db {:communities/all-addresses-to-reveal {community-id #{"0xA" "0xB" "0xC"}}
                 :communities/all-airdrop-addresses   {community-id "0xB"}
-                :wallet                              {:accounts {"0xB" {:address "0xB" :position 0}
-                                                                 "0xA" {:address "0xA" :position 1}
-                                                                 "0xC" {:address "0xC" :position 2}}}}}
+                :wallet                              {:accounts {"0xB" {:address   "0xB"
+                                                                        :operable? true
+                                                                        :position  0}
+                                                                 "0xA" {:address   "0xA"
+                                                                        :operable? true
+                                                                        :position  1}
+                                                                 "0xC" {:address   "0xC"
+                                                                        :operable? true
+                                                                        :position  2}}}}}
           addresses-to-reveal ["0xA" "0xC"]]
       (is (match?
            {:db {:communities/all-addresses-to-reveal
@@ -52,9 +58,9 @@
   (testing "sets flag from false -> true will mark all addresses to be revealed"
     (let [cofx                {:db
                                {:wallet
-                                {:accounts {"0xB" {:address "0xB" :position 0}
-                                            "0xA" {:address "0xA" :position 1}
-                                            "0xC" {:address "0xC" :position 2}}}
+                                {:accounts {"0xB" {:address "0xB" :operable? true :position 0}
+                                            "0xA" {:address "0xA" :operable? true :position 1}
+                                            "0xC" {:address "0xC" :operable? true :position 2}}}
                                 :communities/all-addresses-to-reveal {community-id #{"0xA"}}
                                 :communities/selected-share-all-addresses {community-id false}}}
           addresses-to-reveal #{"0xA" "0xB" "0xC"}]

--- a/src/status_im/contexts/communities/actions/addresses_for_permissions/view.cljs
+++ b/src/status_im/contexts/communities/actions/addresses_for_permissions/view.cljs
@@ -260,7 +260,7 @@
 
         can-edit-addresses? (rf/sub [:communities/can-edit-shared-addresses? id])
 
-        wallet-accounts (rf/sub [:wallet/accounts-without-watched-accounts])
+        wallet-accounts (rf/sub [:wallet/operable-accounts-without-watched-accounts])
         unmodified-addresses-to-reveal (rf/sub [:communities/addresses-to-reveal id])
         [addresses-to-reveal set-addresses-to-reveal] (rn/use-state unmodified-addresses-to-reveal)
 

--- a/src/status_im/contexts/communities/events.cljs
+++ b/src/status_im/contexts/communities/events.cljs
@@ -154,7 +154,7 @@
 (defn update-previous-permission-addresses
   [{:keys [db]} [community-id]]
   (when community-id
-    (let [accounts                      (utils/sorted-non-watch-only-accounts db)
+    (let [accounts                      (utils/sorted-operable-non-watch-only-accounts db)
           selected-permission-addresses (get-in db
                                                 [:communities community-id
                                                  :selected-permission-addresses])
@@ -198,7 +198,7 @@
   [{:keys [db]} [community-id]]
   (let [share-all-addresses?      (get-in db [:communities community-id :share-all-addresses?])
         next-share-all-addresses? (not share-all-addresses?)
-        accounts                  (utils/sorted-non-watch-only-accounts db)
+        accounts                  (utils/sorted-operable-non-watch-only-accounts db)
         addresses                 (set (map :address accounts))]
     {:db (update-in db
                     [:communities community-id]

--- a/src/status_im/contexts/communities/overview/events.cljs
+++ b/src/status_im/contexts/communities/overview/events.cljs
@@ -51,7 +51,7 @@
 
 (rf/reg-event-fx :communities/check-permissions-to-join-community-with-all-addresses
  (fn [{:keys [db]} [community-id]]
-   (let [accounts  (utils/sorted-non-watch-only-accounts db)
+   (let [accounts  (utils/sorted-operable-non-watch-only-accounts db)
          addresses (set (map :address accounts))]
      {:db            (assoc-in db [:communities/permissions-check community-id :checking?] true)
       :json-rpc/call [{:method "wakuext_checkPermissionsToJoinCommunity"

--- a/src/status_im/contexts/communities/utils.cljs
+++ b/src/status_im/contexts/communities/utils.cljs
@@ -17,4 +17,5 @@
   (->> (get-in db [:wallet :accounts])
        (vals)
        (remove :watch-only?)
+       (filter :operable?)
        (sort-by :position)))

--- a/src/status_im/contexts/communities/utils.cljs
+++ b/src/status_im/contexts/communities/utils.cljs
@@ -12,7 +12,7 @@
      constants/community-token-permission-become-member       :t/member
      fallback-to)))
 
-(defn sorted-non-watch-only-accounts
+(defn sorted-operable-non-watch-only-accounts
   [db]
   (->> (get-in db [:wallet :accounts])
        (vals)

--- a/src/status_im/contexts/wallet/data_store.cljs
+++ b/src/status_im/contexts/wallet/data_store.cljs
@@ -23,6 +23,7 @@
 (defn add-keys-to-account
   [account]
   (-> account
+      (assoc :operable? (not= (:operable account) :no))
       (assoc :watch-only? (= (:type account) :watch))
       (assoc :default-account? (:wallet account))))
 
@@ -67,7 +68,7 @@
                         :color                    :colorId})
       (update :prodPreferredChainIds chain-ids-set->string)
       (update :testPreferredChainIds chain-ids-set->string)
-      (dissoc :watch-only? :default-account? :tokens :collectibles)))
+      (dissoc :watch-only? :default-account? :operable? :tokens :collectibles)))
 
 (defn- rpc->balances-per-chain
   [token]

--- a/src/status_im/contexts/wallet/events_test.cljs
+++ b/src/status_im/contexts/wallet/events_test.cljs
@@ -40,6 +40,7 @@
    :color :purple
    :wallet true
    :default-account? true
+   :operable? true
    :name "Ethereum account"
    :type :generated
    :chat false

--- a/src/status_im/contexts/wallet/sheets/account_options/view.cljs
+++ b/src/status_im/contexts/wallet/sheets/account_options/view.cljs
@@ -108,9 +108,8 @@
   []
   (let [options-height (reagent/atom 0)]
     (fn []
-      (let [theme (quo.theme/use-theme)
-            accounts (rf/sub
-                      [:wallet/fully-or-partially-operable-accounts-without-current-viewing-account])
+      (let [theme                  (quo.theme/use-theme)
+            accounts               (rf/sub [:wallet/operable-accounts-without-current-viewing-account])
             show-account-selector? (pos? (count accounts))]
         [:<>
          (when show-account-selector?

--- a/src/status_im/contexts/wallet/sheets/select_account/view.cljs
+++ b/src/status_im/contexts/wallet/sheets/select_account/view.cljs
@@ -20,7 +20,7 @@
 (defn view
   []
   (let [selected-account-address (rf/sub [:wallet/current-viewing-account-address])
-        accounts (rf/sub [:wallet/fully-or-partially-operable-accounts-without-watched-accounts])]
+        accounts                 (rf/sub [:wallet/operable-accounts-without-watched-accounts])]
     [:<>
      [quo/drawer-top {:title (i18n/label :t/select-account)}]
      [gesture/flat-list

--- a/src/status_im/subs/community/account_selection.cljs
+++ b/src/status_im/subs/community/account_selection.cljs
@@ -53,7 +53,7 @@
 
 (re-frame/reg-sub :communities/accounts-to-reveal
  (fn [[_ community-id]]
-   [(re-frame/subscribe [:wallet/accounts-without-watched-accounts])
+   [(re-frame/subscribe [:wallet/operable-accounts-without-watched-accounts])
     (re-frame/subscribe [:communities/addresses-to-reveal community-id])])
  (fn [[accounts addresses] _]
    (filter #(contains? addresses (:address %))
@@ -61,7 +61,7 @@
 
 (re-frame/reg-sub :communities/airdrop-account
  (fn [[_ community-id]]
-   [(re-frame/subscribe [:wallet/accounts-without-watched-accounts])
+   [(re-frame/subscribe [:wallet/operable-accounts-without-watched-accounts])
     (re-frame/subscribe [:communities/airdrop-address community-id])])
  (fn [[accounts airdrop-address] _]
    (->> accounts

--- a/src/status_im/subs/community/account_selection_test.cljs
+++ b/src/status_im/subs/community/account_selection_test.cljs
@@ -24,10 +24,10 @@
 
 (h/deftest-sub :communities/airdrop-account
   [sub-name]
-  (let [airdrop-account {:address "0xA" :position 1}]
+  (let [airdrop-account {:address "0xA" :operable? true :position 1}]
     (reset! rf-db/app-db
       {:communities/all-airdrop-addresses {community-id "0xA"}
-       :wallet                            {:accounts {"0xB" {:address "0xB" :position 0}
+       :wallet                            {:accounts {"0xB" {:address "0xB" :operable? true :position 0}
                                                       "0xA" airdrop-account}}})
 
     (is (match? airdrop-account (rf/sub [sub-name community-id])))))
@@ -36,9 +36,11 @@
   [sub-name]
   (reset! rf-db/app-db
     {:communities/all-addresses-to-reveal {community-id #{"0xC" "0xB"}}
-     :wallet                              {:accounts {"0xB" {:address "0xB" :position 0}
-                                                      "0xA" {:address "0xA" :position 1}
-                                                      "0xC" {:address "0xC" :position 2}}}})
+     :wallet                              {:accounts {"0xB" {:address "0xB" :operable? true :position 0}
+                                                      "0xA" {:address "0xA" :operable? true :position 1}
+                                                      "0xC" {:address   "0xC"
+                                                             :operable? true
+                                                             :position  2}}}})
 
   (is (match? [{:address "0xB" :position 0}
                {:address "0xC" :position 2}]

--- a/src/status_im/subs/wallet/wallet.cljs
+++ b/src/status_im/subs/wallet/wallet.cljs
@@ -454,25 +454,23 @@
  (fn [accounts]
    (remove :watch-only? accounts)))
 
-(defn- keep-fully-or-partially-operable-accounts
+(defn- keep-operable-accounts
   [accounts]
-  (filter (fn fully-or-partially-operable? [{:keys [operable]}]
-            (#{:fully :partially} operable))
-          accounts))
+  (filter :operable? accounts))
 
 (rf/reg-sub
- :wallet/fully-or-partially-operable-accounts-without-current-viewing-account
+ :wallet/operable-accounts-without-current-viewing-account
  :<- [:wallet/accounts-without-current-viewing-account]
- keep-fully-or-partially-operable-accounts)
+ keep-operable-accounts)
 
 (rf/reg-sub
- :wallet/fully-or-partially-operable-accounts-without-watched-accounts
+ :wallet/operable-accounts-without-watched-accounts
  :<- [:wallet/accounts-without-watched-accounts]
- keep-fully-or-partially-operable-accounts)
+ keep-operable-accounts)
 
 (rf/reg-sub
  :wallet/accounts-with-current-asset
- :<- [:wallet/fully-or-partially-operable-accounts-without-watched-accounts]
+ :<- [:wallet/operable-accounts-without-watched-accounts]
  :<- [:wallet/wallet-send-token-symbol]
  :<- [:wallet/wallet-send-token]
  (fn [[accounts token-symbol token]]

--- a/src/status_im/subs/wallet/wallet_test.cljs
+++ b/src/status_im/subs/wallet/wallet_test.cljs
@@ -224,7 +224,7 @@
            (assoc-in [:wallet :accounts] accounts)
            (assoc-in [:wallet :networks] network-data)))
     (is
-     (=
+     (match?
       (list {:path                      "m/44'/60'/0'/0/0"
              :emoji                     "😃"
              :key-uid                   "0x2f5ea39"
@@ -315,30 +315,30 @@
 
     (let [result (rf/sub [sub-name])]
       (is
-       (= {:path                      "m/44'/60'/0'/0/0"
-           :emoji                     "😃"
-           :key-uid                   "0x2f5ea39"
-           :address                   "0x1"
-           :wallet                    false
-           :name                      "Account One"
-           :type                      :generated
-           :watch-only?               false
-           :operable?                 true
-           :chat                      false
-           :test-preferred-chain-ids  #{5 420 421613}
-           :color                     :blue
-           :hidden                    false
-           :prod-preferred-chain-ids  #{1 10 42161}
-           :network-preferences-names #{:mainnet :arbitrum :optimism}
-           :position                  0
-           :clock                     1698945829328
-           :created-at                1698928839000
-           :operable                  :fully
-           :mixedcase-address         "0x7bcDfc75c431"
-           :public-key                "0x04371e2d9d66b82f056bc128064"
-           :removed                   false
-           :tokens                    tokens-0x1}
-          (dissoc result :balance :formatted-balance)))
+       (match? {:path                      "m/44'/60'/0'/0/0"
+                :emoji                     "😃"
+                :key-uid                   "0x2f5ea39"
+                :address                   "0x1"
+                :wallet                    false
+                :name                      "Account One"
+                :type                      :generated
+                :watch-only?               false
+                :operable?                 true
+                :chat                      false
+                :test-preferred-chain-ids  #{5 420 421613}
+                :color                     :blue
+                :hidden                    false
+                :prod-preferred-chain-ids  #{1 10 42161}
+                :network-preferences-names #{:mainnet :arbitrum :optimism}
+                :position                  0
+                :clock                     1698945829328
+                :created-at                1698928839000
+                :operable                  :fully
+                :mixedcase-address         "0x7bcDfc75c431"
+                :public-key                "0x04371e2d9d66b82f056bc128064"
+                :removed                   false
+                :tokens                    tokens-0x1}
+               (dissoc result :balance :formatted-balance)))
 
       (is (money/equal-to (:balance result) (money/bignumber 3250)))
       (is (match? (:formatted-balance result) "$3250.00")))))
@@ -376,7 +376,7 @@
            (assoc-in [:wallet :current-viewing-account-address] "0x2")
            (assoc-in [:wallet :networks] network-data)))
     (is
-     (=
+     (match?
       (list
        {:path                      "m/44'/60'/0'/0/0"
         :emoji                     "😃"
@@ -434,7 +434,7 @@
            (assoc-in [:wallet :accounts] accounts)
            (assoc-in [:wallet :networks] network-data)))
     (is
-     (=
+     (match?
       (list
        {:path                      "m/44'/60'/0'/0/0"
         :emoji                     "😃"
@@ -592,19 +592,19 @@
            (assoc-in [:wallet :accounts] accounts)
            (assoc-in [:wallet :networks] network-data)))
     (is
-     (= [(-> accounts
-             (get "0x1")
-             (assoc :customization-color :blue)
-             (assoc :network-preferences-names #{:mainnet :arbitrum :optimism}))
-         (-> accounts
-             (get "0x2")
-             (assoc :customization-color :purple)
-             (assoc :network-preferences-names #{:mainnet :arbitrum :optimism}))
-         (-> accounts
-             (get "0x3")
-             (assoc :customization-color :magenta)
-             (assoc :network-preferences-names #{}))]
-        (rf/sub [sub-name])))))
+     (match? [(-> accounts
+                  (get "0x1")
+                  (assoc :customization-color :blue)
+                  (assoc :network-preferences-names #{:mainnet :arbitrum :optimism}))
+              (-> accounts
+                  (get "0x2")
+                  (assoc :customization-color :purple)
+                  (assoc :network-preferences-names #{:mainnet :arbitrum :optimism}))
+              (-> accounts
+                  (get "0x3")
+                  (assoc :customization-color :magenta)
+                  (assoc :network-preferences-names #{}))]
+             (rf/sub [sub-name])))))
 
 (h/deftest-sub :wallet/watch-only-accounts
   [sub-name]
@@ -614,10 +614,10 @@
            (assoc-in [:wallet :accounts] accounts)
            (assoc-in [:wallet :networks] network-data)))
     (is
-     (= [(-> accounts
-             (get "0x3")
-             (assoc :network-preferences-names #{}))]
-        (rf/sub [sub-name])))))
+     (match? [(-> accounts
+                  (get "0x3")
+                  (assoc :network-preferences-names #{}))]
+             (rf/sub [sub-name])))))
 
 (def chat-account
   {:path     "m/43'/60'/1581'/0'/0"
@@ -827,9 +827,7 @@
   (testing "returns local suggestions:"
     (swap! rf-db/app-db
       #(assoc-in % [:wallet :ui :search-address :local-suggestions] local-suggestions))
-    (is
-     (= local-suggestions
-        (rf/sub [sub-name])))))
+    (is (match? local-suggestions (rf/sub [sub-name])))))
 
 (h/deftest-sub :wallet/valid-ens-or-address?
   [sub-name]

--- a/src/status_im/subs/wallet/wallet_test.cljs
+++ b/src/status_im/subs/wallet/wallet_test.cljs
@@ -16,10 +16,12 @@
   {:0x1 {:tokens                    [{:symbol "ETH"} {:symbol "SNT"}]
          :network-preferences-names #{}
          :customization-color       nil
+         :operable?                 true
          :operable                  :fully}
    :0x2 {:tokens                    [{:symbol "SNT"}]
          :network-preferences-names #{}
          :customization-color       nil
+         :operable?                 true
          :operable                  :partially}})
 
 (def tokens-0x1
@@ -106,6 +108,7 @@
           :name                     "Account One"
           :type                     :generated
           :watch-only?              false
+          :operable?                true
           :chat                     false
           :test-preferred-chain-ids #{5 420 421613}
           :color                    :blue
@@ -127,6 +130,7 @@
           :name                     "Account Two"
           :type                     :generated
           :watch-only?              false
+          :operable?                true
           :chat                     false
           :test-preferred-chain-ids #{5 420 421613}
           :color                    :purple
@@ -148,6 +152,7 @@
           :name                     "Watched Account 1"
           :type                     :watch
           :watch-only?              true
+          :operable?                true
           :chat                     false
           :test-preferred-chain-ids #{0}
           :color                    :magenta
@@ -228,6 +233,7 @@
              :name                      "Account One"
              :type                      :generated
              :watch-only?               false
+             :operable?                 true
              :chat                      false
              :test-preferred-chain-ids  #{5 420 421613}
              :color                     :blue
@@ -250,6 +256,7 @@
              :name                      "Account Two"
              :type                      :generated
              :watch-only?               false
+             :operable?                 true
              :chat                      false
              :test-preferred-chain-ids  #{5 420 421613}
              :color                     :purple
@@ -272,6 +279,7 @@
              :name                      "Watched Account 1"
              :type                      :watch
              :watch-only?               true
+             :operable?                 true
              :chat                      false
              :test-preferred-chain-ids  #{0}
              :color                     :magenta
@@ -315,6 +323,7 @@
            :name                      "Account One"
            :type                      :generated
            :watch-only?               false
+           :operable?                 true
            :chat                      false
            :test-preferred-chain-ids  #{5 420 421613}
            :color                     :blue
@@ -367,52 +376,55 @@
            (assoc-in [:wallet :current-viewing-account-address] "0x2")
            (assoc-in [:wallet :networks] network-data)))
     (is
-     (= (list
-         {:path                      "m/44'/60'/0'/0/0"
-          :emoji                     "😃"
-          :key-uid                   "0x2f5ea39"
-          :address                   "0x1"
-          :wallet                    false
-          :name                      "Account One"
-          :type                      :generated
-          :watch-only?               false
-          :chat                      false
-          :test-preferred-chain-ids  #{5 420 421613}
-          :color                     :blue
-          :hidden                    false
-          :prod-preferred-chain-ids  #{1 10 42161}
-          :network-preferences-names #{:mainnet :arbitrum :optimism}
-          :position                  0
-          :clock                     1698945829328
-          :created-at                1698928839000
-          :operable                  :fully
-          :mixedcase-address         "0x7bcDfc75c431"
-          :public-key                "0x04371e2d9d66b82f056bc128064"
-          :removed                   false
-          :tokens                    tokens-0x1}
-         {:path                      ""
-          :emoji                     "🎉"
-          :key-uid                   "0x2f5ea39"
-          :address                   "0x3"
-          :wallet                    false
-          :name                      "Watched Account 1"
-          :type                      :watch
-          :watch-only?               true
-          :chat                      false
-          :test-preferred-chain-ids  #{0}
-          :color                     :magenta
-          :hidden                    false
-          :prod-preferred-chain-ids  #{0}
-          :network-preferences-names #{}
-          :position                  2
-          :clock                     1698945829328
-          :created-at                1698928839000
-          :operable                  :fully
-          :mixedcase-address         "0x7bcDfc75c431"
-          :public-key                "0x"
-          :removed                   false
-          :tokens                    tokens-0x3})
-        (rf/sub [sub-name])))))
+     (=
+      (list
+       {:path                      "m/44'/60'/0'/0/0"
+        :emoji                     "😃"
+        :key-uid                   "0x2f5ea39"
+        :address                   "0x1"
+        :wallet                    false
+        :name                      "Account One"
+        :type                      :generated
+        :watch-only?               false
+        :operable?                 true
+        :chat                      false
+        :test-preferred-chain-ids  #{5 420 421613}
+        :color                     :blue
+        :hidden                    false
+        :prod-preferred-chain-ids  #{1 10 42161}
+        :network-preferences-names #{:mainnet :arbitrum :optimism}
+        :position                  0
+        :clock                     1698945829328
+        :created-at                1698928839000
+        :operable                  :fully
+        :mixedcase-address         "0x7bcDfc75c431"
+        :public-key                "0x04371e2d9d66b82f056bc128064"
+        :removed                   false
+        :tokens                    tokens-0x1}
+       {:path                      ""
+        :emoji                     "🎉"
+        :key-uid                   "0x2f5ea39"
+        :address                   "0x3"
+        :wallet                    false
+        :name                      "Watched Account 1"
+        :type                      :watch
+        :watch-only?               true
+        :operable?                 true
+        :chat                      false
+        :test-preferred-chain-ids  #{0}
+        :color                     :magenta
+        :hidden                    false
+        :prod-preferred-chain-ids  #{0}
+        :network-preferences-names #{}
+        :position                  2
+        :clock                     1698945829328
+        :created-at                1698928839000
+        :operable                  :fully
+        :mixedcase-address         "0x7bcDfc75c431"
+        :public-key                "0x"
+        :removed                   false
+        :tokens                    tokens-0x3})
+      (rf/sub [sub-name])))))
 
 (h/deftest-sub :wallet/accounts-without-watched-accounts
   [sub-name]
@@ -432,6 +444,7 @@
         :name                      "Account One"
         :type                      :generated
         :watch-only?               false
+        :operable?                 true
         :chat                      false
         :test-preferred-chain-ids  #{5 420 421613}
         :color                     :blue
@@ -455,6 +468,7 @@
         :name                      "Account Two"
         :type                      :generated
         :watch-only?               false
+        :operable?                 true
         :chat                      false
         :test-preferred-chain-ids  #{5 420 421613}
         :color                     :purple
@@ -484,6 +498,7 @@
                   [{:tokens                    [{:symbol "ETH"} {:symbol "SNT"}]
                     :network-preferences-names #{}
                     :customization-color       nil
+                    :operable?                 true
                     :operable                  :fully}]))))
 
   (testing "returns the accounts list with the current asset using token"
@@ -496,6 +511,7 @@
                   [{:tokens                    [{:symbol "ETH"} {:symbol "SNT"}]
                     :network-preferences-names #{}
                     :customization-color       nil
+                    :operable?                 true
                     :operable                  :fully}]))))
 
   (testing

